src/options.ts

```diff
@@ -1,9 +1,10 @@
-import { BeautifierOptions, OptionValues } from "unibeautify";
-
+import {BeautifierOptions, OptionValues} from "unibeautify";
```

Comment:
I prefer to have spaces in here.
https://unibeautify.com/docs/option-object-curly-spacing.html#javascript